### ISSUE TITLE
IDToRemove should be send as pointer

### ISSUE THREAD

**Issue by troian** [web-ridge/gqlgen-sqlboiler#32]
```
{"level":"error","error":"obj type should be *Type, *[]Type, or *[]*Type but was \"[]boilergql.RemovedStringID\"","time":"2020-05-22T20:19:54Z","message":"could not delete dealSearchStates"}
```

Generated code is
```go
	var IDsToRemove []boilergql.RemovedStringID
	if err := dm.UserSocials(mods...).Bind(ctx, r.db, IDsToRemove); err != nil {
		log.Error().Err(err).Msg(publicUserSocialBatchDeleteError)
		return nil, errors.New(publicUserSocialBatchDeleteError)
	}
```

expected code
```go
	var IDsToRemove []boilergql.RemovedStringID
	if err := dm.UserSocials(mods...).Bind(ctx, r.db, &IDsToRemove); err != nil {
		log.Error().Err(err).Msg(publicUserSocialBatchDeleteError)
		return nil, errors.New(publicUserSocialBatchDeleteError)
	}
```

**Comment by RichardLindhout** [comment]
Can you make a PR for this? Code is in resolver.gotpl.

I don't have the use-case yet. Ideally we should automatically test the output if works end-to-end.

**Comment by RichardLindhout** [comment]
Fixed in v2.1.2

**Comment by troian** [comment]
Sorry, I had a first relaxing weekend for a long while.
Thanks for the fix!
